### ISSUE TITLE
UnicodeEncodeError when uploading reports data

### ISSUE THREAD

**Issue by vinta** [codecov/codecov-python#35]
``` py
$ codecov --token=MY_CODECOV_TOKEN
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.4.2/bin/codecov", line 9, in <module>
    load_entry_point('codecov==1.2.3', 'console_scripts', 'codecov')()
  File "/home/travis/virtualenv/python3.4.2/lib/python3.4/site-packages/codecov/__init__.py", line 325, in cli
    data, codecov = main()
  File "/home/travis/virtualenv/python3.4.2/lib/python3.4/site-packages/codecov/__init__.py", line 318, in main
    data = upload(url=codecov.url, branch=codecov.branch, commit=codecov.commit, token=codecov.token, env=codecov.env, slug=codecov.slug, **defaults)
  File "/home/travis/virtualenv/python3.4.2/lib/python3.4/site-packages/codecov/__init__.py", line 137, in upload
    result = requests.post(url, data=reports)
  File "/home/travis/virtualenv/python3.4.2/lib/python3.4/site-packages/requests/api.py", line 88, in post
    return request('post', url, data=data, **kwargs)
  File "/home/travis/virtualenv/python3.4.2/lib/python3.4/site-packages/requests/api.py", line 44, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/travis/virtualenv/python3.4.2/lib/python3.4/site-packages/requests/sessions.py", line 456, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/travis/virtualenv/python3.4.2/lib/python3.4/site-packages/requests/sessions.py", line 559, in send
    r = adapter.send(request, **kwargs)
  File "/home/travis/virtualenv/python3.4.2/lib/python3.4/site-packages/requests/adapters.py", line 327, in send
    timeout=timeout
  File "/home/travis/virtualenv/python3.4.2/lib/python3.4/site-packages/requests/packages/urllib3/connectionpool.py", line 493, in urlopen
    body=body, headers=headers)
  File "/home/travis/virtualenv/python3.4.2/lib/python3.4/site-packages/requests/packages/urllib3/connectionpool.py", line 291, in _make_request
    conn.request(method, url, **httplib_request_kw)
  File "/opt/python/3.4.2/lib/python3.4/http/client.py", line 1090, in request
    self._send_request(method, url, body, headers)
  File "/opt/python/3.4.2/lib/python3.4/http/client.py", line 1127, in _send_request
    body = body.encode('iso-8859-1')
UnicodeEncodeError: 'latin-1' codec can't encode characters in position 21994-21999: ordinal not in range(256)
```

**Comment by stevepeak** [comment]
Hey thank you for reporting this. Can you do me a favor and check our the upcoming changes to see if it fixes this issue?

```sh
pip uninstall -y codecov
pip install git+git://github.com/codecov/codecov-python.git@wip/1.3.0
codecov --token=MY_CODECOV_TOKEN
```

If this fails I would like to know what the coverage file is that failed to see if I can parse out these values.

Thanks!

**Comment by vinta** [comment]
It still fails, I has emailed you the coverage file.

**Comment by stevepeak** [comment]
Thank you for the email. Seems this file is not an actual coverage report and should be ignored. I'll be releases our v1.3.0 soon which fixes this issue via 73b5a2e

**Comment by thomasrockhu** [comment]
Closing due to age of issue.
